content/600-about/01-about-the-docs.mdx

```diff
@@ -4,11 +4,22 @@ metaTitle: 'About the Prisma documentation'
 metaDescription: 'This page gives a meta-overview about different topics that are relevant to better understand and navigate the Prisma documentation.'
 ---
 
-## Overview
+<TopBlock>
+
+`User` and `Post` are the canonical models that are being used throughout the Prisma docs. This page gives some context on why these have been selected and how to interpret them.
```

Comment:
Uh, this is not actually correct: The page is about much more than just that. Just one section is about this.